lib/ansible/modules/cloud/vmware/vmware_vspan_session.py

```diff
@@ -82,18 +83,18 @@
         description:
             - Source port for which transmitted packets are mirrored.
         required: False
-        type: str
+        type: list
```

Comment:
correct me if I'm wrong, but if someone continue to pass a string (e.g: '13'), the module will cast the string to a list. And you will end-up with a list of character. So `['1', '3']` with my example. This is not what we want.

My conclusion is that this changeset will silently break the users playbooks.